src/main/java/Activities.java

```diff
@@ -0,0 +1,33 @@
+public class Activities {
+    private final String Activity;
+    private int status = 0; // 0 represent undone: 1 represents done
+
+    public Activities(String activity) {
+        Activity = activity;
+    }
+    public static Activities newActivity(String activity){
+        return new Activities(activity);
+    }
+
+    public void doIt() {
+        status = 1;
+    }
+
+    public String getActivity(){
+        return Activity;
+    }
+    public int isDone(){
```

Comment:
Should there be spaces between your methods? I noticed that there is a little inconsistency in several other places too.